plugin/src/main/java/org/opensearch/ml/breaker/DiskCircuitBreaker.java

```diff
@@ -20,6 +23,7 @@ public class DiskCircuitBreaker extends ThresholdCircuitBreaker<Long> {
     private static final String ML_DISK_CB = "Disk Circuit Breaker";
     public static final long DEFAULT_DISK_SHORTAGE_THRESHOLD = 5L;
     private static final long GB = 1024 * 1024 * 1024;
+    private volatile float diskShortageThreshold = 5L;
```

Comment:
Why is a float variable here given a long constant?

Reply:
I will fix it.